### PR TITLE
MC-12234: Adding some missing attributes to the get and list for orga…

### PR DESCRIPTION
**Pull request by evanluc**
- I added custom domain to the optional attributes in the update organization. As well as missing updatable fields. There are no required fields for the update also, so I moved the required attributes to optional.
- There was a bunch of missing attributes for the Organization`get` and `list`. I added these.

### CODE REVIEW

File: source/includes/administration/_organizations.md [hunk 1]
```diff
@@ -69,7 +69,13 @@ Attributes | &nbsp;
 `environments`<br/>*Array[[Environment](#administration-environments)]* | The environments belonging to the organization.<br/>*includes*: `id`
 `roles`<br/>*Array[[Role](#administration-roles)]* | The system and environments roles belonging to the organization.<br/>*includes*: `id`
 `serviceConnections`<br/>*Array[[ServiceConnection](#administration-service-connections)]* | The services for which the organization is allowed to provision resources.<br/>*includes*: `id`,`serviceCode`
+`resourceCommitments`</br>*Array[[ResourceCommitment](#administration-retrieve-a-resource-commitment)]* | The resource commitments applied on the organization.
 `users`<br/>*Array[[User](#administration-users)]* | The users of the organization.<br/>*includes*: `id`
+`notes`<br/>*string* | Organization notes.
+`isDbAuthentication`<br/>*boolean* | Whether or not the organization supports db authentication.
+`isLdapAuthentication`<br/>*boolean* | Whether or ldap authentication is enabled on this organization.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_organizations.md]: `Whether or not`

File: source/includes/administration/_organizations.md [hunk 2]
```diff
@@ -136,7 +142,13 @@ Attributes | &nbsp;
...
```
Comment (GuillaumeVialle) [inline source/includes/administration/_organizations.md]: same here

File: source/includes/administration/_organizations.md [hunk 3]
```diff
@@ -209,14 +221,18 @@ curl -X PUT "https://cloudmc_endpoint/v1/organizations/03bc22bd-adc4-46b8-988d-a
...
-
-Optional | &nbsp;
----- | ----
 `serviceConnections`<br/>Array[[ServiceConnection](#administration-service-connections)] | A list of service connections for which the organization may provision resources. The caller must have access to all connections that are provided. **NB :** Service connection access may be added but not revoked at this time.<br/>*required :* `id`
+`tags`<br/>*Array[string]* | Tags associated to the organization.
+`resourceCommitments`</br>*Array[[ResourceCommitment](#administration-retrieve-a-resource-commitment)]* | The resource commitments applied on the organization.
+`users`<br/>*Array[[User](#administration-users)]* | The users of the organization.<br/>*required* : `id`
+`notes`<br/>*string* | Organization notes. Must have the `Organization metadata: Manage` permission.
+`isDbAuthentication`<br/>*boolean* | Whether or not the organization supports db authentication.
+`isLdapAuthentication`<br/>*boolean* | Whether or ldap authentication is enabled on this organization.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_organizations.md]: same here

File: source/includes/administration/_organizations.md [hunk 4]
```diff
@@ -69,7 +69,13 @@ Attributes | &nbsp;
 `environments`<br/>*Array[[Environment](#administration-environments)]* | The environments belonging to the organization.<br/>*includes*: `id`
 `roles`<br/>*Array[[Role](#administration-roles)]* | The system and environments roles belonging to the organization.<br/>*includes*: `id`
 `serviceConnections`<br/>*Array[[ServiceConnection](#administration-service-connections)]* | The services for which the organization is allowed to provision resources.<br/>*includes*: `id`,`serviceCode`
+`resourceCommitments`</br>*Array[[ResourceCommitment](#administration-retrieve-a-resource-commitment)]* | The resource commitments applied on the organization.
 `users`<br/>*Array[[User](#administration-users)]* | The users of the organization.<br/>*includes*: `id`
+`notes`<br/>*string* | Organization notes.
+`isDbAuthentication`<br/>*boolean* | Whether or not the organization supports db authentication.
```
Comment (patrickspensieri) [inline source/includes/administration/_organizations.md]: Should be _LDAP_ and _database_ authentication

**Review by patrickspensieri — APPROVED** [review]

File: source/includes/administration/_organizations.md (hunk 1 above)
Comment (evanluc) [reply source/includes/administration/_organizations.md]: 4cdafc3c34eaae06cf9dc2a14eda26a696e61337

File: source/includes/administration/_organizations.md (hunk 2 above)
Comment (evanluc) [reply source/includes/administration/_organizations.md]: 4cdafc3c34eaae06cf9dc2a14eda26a696e61337

File: source/includes/administration/_organizations.md (hunk 3 above)
Comment (evanluc) [reply source/includes/administration/_organizations.md]: 4cdafc3c34eaae06cf9dc2a14eda26a696e61337

File: source/includes/administration/_organizations.md (hunk 4 above)
Comment (evanluc) [reply source/includes/administration/_organizations.md]: 4cdafc3c34eaae06cf9dc2a14eda26a696e61337

**Review by GuillaumeVialle — APPROVED** [review]

**Pull request merged**